pkg/commands/template.go

```diff
@@ -0,0 +1,43 @@
+package commands
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/oam-dev/kubevela/api/types"
+	cmdutil "github.com/oam-dev/kubevela/pkg/commands/util"
+	mycue "github.com/oam-dev/kubevela/pkg/cue"
+)
+
+func NewTemplateCommand(c types.Args, ioStream cmdutil.IOStreams) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:                   "template",
```

Comment:
What does this command used for?

Reply:
who is your target user to create a template?

I assume the template is a system level configure, or at least defined in WorkloadDefinition/TraitDefinition level?

Reply:
I assume all template related stuff will be put under this command.
Currently just to show context def.

Reply:
> I assume the template is a system level configure, or at least defined in WorkloadDefinition/TraitDefinition level?

Can you clarify or be more specific?